modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -359,6 +368,16 @@ protected void containerIsStarting(final InspectContainerResponse containerInfo)
         }
     }
 
+    @Override
+    protected void containerIsStarted(InspectContainerResponse containerInfo, boolean reused) {
+        if (reused) {
+            logger()
+                .info("Couchbase container reused and ready! UI available at http://{}:{}", getHost(), getMappedPort(MGMT_PORT));
+        } else {
+            this.containerIsStarted(containerInfo);
+        }
```

Comment:
log is redundant. Implementation should be the same as suggested at `containerIsStarting`